backend-lib/build.gradle.kts

```diff
@@ -94,10 +102,37 @@ cargo {
     }
 }
 
+protobuf {
+    protoc {
+        artifact = libs.protoc.compiler.get().asCoordinateString()
+    }
+    plugins {
+        id("java") {
+            artifact = libs.protoc.gen.java.get().asCoordinateString()
+        }
+    }
+    generateProtoTasks {
+        all().forEach {
+            it.plugins {
+                id("java") {
+                    option("lite")
+                }
+            }
+            it.builtins {
+                id("kotlin") {
+                    option("lite")
+                }
+            }
+        }
+    }
+}
+
 dependencies {
     api(projects.lightwalletClientLib)
 
     implementation(libs.androidx.annotation)
+    // TODO: We don't need gRPC here, only protoc
+    implementation(libs.bundles.grpc)
```

Comment:
@HonzaR I just copied stuff from `lightwallet-client-lib/build.gradle.kts` until I got the protobufs to generate; I expect we can refine things here since we only care about the protobufs in `backend-lib`, not gRPC.

Reply:
I just pushed a little improvement of the dependency that we need here.